### ISSUE TITLE
Don´t works today

### ISSUE THREAD

**Issue by antxiko** [IkerGarcia/PocketInstaller#30]
It could be possible to update it for making it work today? Thanks in advance

**Comment by CaptainZalo** [comment]
I'll look into it when I have some spare time. The repos haven't been updated in about 4 years, so ymmv.

**Comment by Raymond-Epitech** [comment]
Yup , the big problem is the ipi.sh that you cant download anymore and the fact that some of the links/dependencies are dead... That's what you get for using dying hardware ig...

**Comment by CaptainZalo** [comment]
There is a lot of action on the CHIP discord. I'll go through on the newly
flashed msinline to see what can be done.

Thanks for your feedback.

ons. 19. jan. 2022, 07:49 skrev Raymond-Epitech ***@***.***>:

**Comment by IkerGarcia** [comment]
I'm sorry to read this but as you can see I've done very little electronics/coding stuff lately as I don't have much time. Moreover, now I mostly use other devices. If CaptainZalo fixes it I will merge no doubt. Thank you all.

**Comment by Raymond-Epitech** [comment]
Thanks alot for your time and consideration though, you guys made the best app! And the fact that you still spend time going out of your way to awnser theses messages is amazing!
